requirements.txt

```diff
@@ -1,6 +1,3 @@
-Django==1.6.5
-argparse==1.2.1
-django-appconf==0.6
-django-compressor==1.4
-six==1.7.2
-wsgiref==0.1.2
+nose
+coverage
+django-nose
```

Comment:
Personally I prefer a file `requirements.test.txt` for tests stuffs. It permits to separate development freezes versions and tests requirements versions. May be we will need several `requirements.2.x-dj1.x.txt` files to cover our big tox matrix. For now it works pretty like that ;)